spec/features/articles_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+describe ArticlesController, type: :feature do
+  let(:user) { create(:user) }
+  before { login user }
+
+  describe '#index GET /articles' do
+    describe 'sort by shipped_at desc' do
+      let!(:today) { create(:article, :with_tags, shipped_at: Date.today) }
+      let!(:yesterday) { create(:article, :with_tags, shipped_at: Date.yesterday) }
+      let(:first_report) { find('#article_index').first('a') }
+
+      before { visit articles_path }
+      it { expect(first_report[:href]).to eq article_path(today) }
+    end
+  end
+
+  describe '#user GET /articles/users/:user_id' do
+    let!(:today) { create(:article, :with_tags, user: user, shipped_at: Date.today) }
```

Comment:
__[RuboCop]__

Rails/Date: Do not use `Date.today` without zone. Use `Time.zone.today` instead.


<!-- Comment by [SideCI](https://sideci.com) -->